app/controllers/carts_controller.rb

```diff
@@ -16,6 +18,13 @@ def index
     @items = @cart.items
   end
 
+  def update
+    @cart.update_item(cart_params)
```

Comment:
Ok makes sense. I'm content with whatever you decide. What about the `before_action`?